x-pack/plugins/cloud_security_posture/server/routes/status/status.test.ts

```diff
@@ -7,6 +7,24 @@
 
 import { calculateIntegrationStatus } from './status';
 import { CSPM_POLICY_TEMPLATE, VULN_MGMT_POLICY_TEMPLATE } from '../../../common/constants';
+import { Installation } from '@kbn/fleet-plugin/common';
+
+const mockInstallation: Installation = {
```

Comment:
looks good, lets add a test case when installation is empty it returns `not-installed`